$NetBSD: patch-ad,v 1.6 2000/08/21 04:00:23 simonb Exp $


Note:  before adding drives to the compatibility list, please test them
       thoroughly.  This means you should be able to read the same
       track off a CD at least 10 times and get identical files out.
       Once you've done this for several CD's, then you can say that
       the drive has been sufficiently tested, and add it to the list.
       



--- cdd_cdcmds.c.orig	Sun May 31 17:13:24 1998
+++ cdd_cdcmds.c	Sun Aug 20 23:45:24 2000
@@ -160,6 +160,46 @@
         mmc_end_cdda
         }
     ,
+        { 
+        "SONY CDU4811",
+        mmc_getcdtoc,
+        mmc_cd_setblocksize,
+        mmc_cdrawread,
+        mmc_cddaread,
+        mmc_begin_cdda,
+        mmc_end_cdda
+        }
+    ,
+        { 
+        "MATSHITA CD-R   CW-7502",
+        mmc_getcdtoc,
+        mmc_cd_setblocksize,
+        mmc_cdrawread,
+        mmc_cddaread,
+        mmc_begin_cdda,
+        mmc_end_cdda
+        }
+    ,
+        { 
+        "MATSHITA CD-R   CW-7582",
+        mmc_getcdtoc,
+        mmc_cd_setblocksize,
+        mmc_cdrawread,
+        mmc_cddaread,
+        mmc_begin_cdda,
+        mmc_end_cdda
+        }
+    ,
+        { 
+        "TEAC CD-532E-A",
+        mmc_getcdtoc,
+        mmc_cd_setblocksize,
+        mmc_cdrawread,
+        mmc_cddaread,
+        mmc_begin_cdda,
+        mmc_end_cdda
+        }
+    ,
         {
         "END-OF-DEVICES",
         unsupported,
